src/store/domains/globalSearch.ts

```diff
@@ -358,7 +358,7 @@ export default class GlobalSearch implements GlobalSearchStoreInterface, Routing
   }
 
   private handleRoutingNotificationForEntityType(value: string) {
-    if (value in Object.keys(EntityType)){
```

Comment:
`x in []` macht kein Sinn, da `in` nur für Objekte funktioniert. Für Arrays muss da der Weg über `includes` gegangen werden. Zudem muss anhand der Values gematched werden.